x-pack/plugins/observability_solution/observability_ai_assistant/server/functions/index.ts

```diff
@@ -62,15 +60,17 @@ export const registerFunctions: RegistrationCallback = async ({
         DO NOT UNDER ANY CIRCUMSTANCES generate ES|QL queries or explain anything about the ES|QL query language yourself.
         DO NOT UNDER ANY CIRCUMSTANCES try to correct an ES|QL query yourself - always use the "query" function for this.
 
+        DO NOT UNDER ANY CIRCUMSTANCES USE ES|QL syntax (\`service.name == "foo"\`) with "kqlFilter" (\`service.name:"foo"\`).
```

Comment:
We are not religious about avoid "not" statements? (honest question)

Reply:
I would like to avoid it, but in some cases it helps. In this case it was a little stubborn in mixing these two up, and it's better now (very unscientifically measured though).